web_src/less/_repository.less

```diff
@@ -1045,6 +1042,17 @@
                     box-shadow: none;
                 }
             }
+
+            .ui.form {
+                .field.footer {
+                    overflow: hidden;
+                    .controls.right {
```

Comment:
```suggestion
                    .controls.right :last-child {
```

Reply:
I'm not sure if it's the right approach, will probably switch buttons margin from right to left side when parent container has `right` class

Reply:
It is the same as yours, no need to use stacked syntax if there is single rule

Reply:
I was just talking to myself while reading those styles.

Reply:
changed in https://github.com/go-gitea/gitea/pull/11139/commits/41a427a3351b5609aa905c9647a9f2f555c01788